PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListItemPresenter.swift

```diff
@@ -38,17 +38,19 @@ private extension Style {
 
 class ItemsListItemPresenter {
     private let item: ItemsListItem
+    private let isDisabled: Bool
 
-    init(item: ItemsListItem) {
+    init(item: ItemsListItem, isDisabled: Bool) {
```

Comment:
looks like in the existing calls, `isDisabled` is predominantly set to `false`: maybe we can add a default value?

Reply:
I think that'd clean up the unit tests a lot, specifically (which is mainly where the additions are).